projects/ngx-location-viewer/src/lib/components/layer-management/layer-management.component.html

```diff
@@ -1,34 +1,21 @@
-<div class="sidebar">
-    <div class="sidebar__header">
-        <button
-            class="a-button-transparent a-button--default has-icon sidebar__close-button"
-            aria-label="Sluiten"
-            (click)="toggleLayermanagement()"
-        >
-            <span class="fa fa-close" aria-hidden="true"></span>
-        </button>
+<h1 class="has-base-font u-text-bold h5">Lagen</h1>
+<div class="u-margin-left-xs">
```

Comment:
Is er een reden waarom je niet gewoon `u-margin` gebruikt op de container? Nu is de padding waarschijnlijk wat onregelmatig, plus verschillende paddings op verschillende plaatsen lokken nogal snel layout issues uit.